src/appleseed/renderer/modeling/bsdf/fresnel.h

```diff
@@ -0,0 +1,118 @@
+
+//
+// This source file is part of appleseed.
+// Visit http://appleseedhq.net/ for additional information and resources.
+//
+// This software is released under the MIT license.
+//
+// Copyright (c) 2010-2013 Francois Beaune, Jupiter Jazz Limited
+// Copyright (c) 2014-2016 Francois Beaune, The appleseedhq Organization
+// Copyright (c) 2014-2016 Esteban Tovagliari, The appleseedhq Organization
```

Comment:
If this is a new file, put only your name:

    // Copyright (c) 2016 Esteban Tovagliari, The appleseedhq Organization